test/test_rosbag/CMakeLists.txt

```diff
@@ -23,6 +23,10 @@ if(CATKIN_ENABLE_TESTING)
     FILENAME test/test_future_version_2.1.bag MD5 ddcb5e8711b16514eb640330ed93c332)
   catkin_download_test_data(download_data_test_rosbag_latched_pub.bag http://download.ros.org/data/rosbag/test_rosbag_latched_pub.bag
     FILENAME test/test_rosbag_latched_pub.bag MD5 ed70b8cbb8a53220434aeb24bccf7342)
+  catkin_download_test_data(download_data_test_clock.bag http://download.ros.org/data/rosbag/test_clock.bag
+    FILENAME test/clock.bag MD5 d81aca4472ad4220b282ecf9e2e557e7)
+  catkin_download_test_data(download_data_test_clock_alive.bag http://tutux.web4all.fr/ros/clock_alive.bag
```

Comment:
Sorry, the file is finally available on ros.org: see http://download.ros.org/data/rosbag/clock_alive.bag

Can you please update the PR to use that URL.